conf/config.neon

```diff
@@ -3,6 +3,7 @@ parameters:
 	excludes_analyse: []
 	autoload_directories: []
 	autoload_files: []
+	custom_extensions: []
```

Comment:
Rename to `fileExtensions` and add `php` there by default so it doesn't have to have any special treatment.